examples/simple-platform/operation.adoc

```diff
@@ -0,0 +1,226 @@
++
+image:img/screen-accounts_app-authorize.png["The authorization screen in
+the Accounts app", 1280, 710, role="frame"]
++
+The `templatePrivate` chain is requesting a read join with the
+`accountsPrivate` chain for the `name` and `email` profile fields. Joins
+can be configured so that only specific fields are shared. Since the
+`email` field does not yet exist, you are given the opportunity to
+create and populate the field.
+
+. Click the **Add email** link, enter an email address, then click the
+**Save** button.
++
+image:img/form-accounts_app-email.png["The email form in the Accounts
+app", 645, 377, role="frame"]
++
+The authorization screen updates, and the **Accept** button becomes
+active:
++
+image:img/form-accounts_app-filled.png["The authorization form in the
+Accounts app", 645, 279, role="frame"]
++
+Using the browser's developer console, you can verify that the
+`accountsPrivate` chain has a new block, and that the email address has
+been added to the profile.
+
+
+. [[step_authorize]]image:img/button-accept.png["The Accept button", 133,
+  48, role="right"]
+  Authorize the request for information by clicking the **Accept**
+  button.
++
+**At this point, you are redirected to the Template app.**
++
+In the Template app and should see a message stating `Complete the cAuth
+loop` with a profile identifier of the form `PROFILE-****`.
++
+image:img/screen-template_app-complete_cauth.png["The Complete the cAuth
+loop screen in the Template app", 1280, 268, role="frame"]
+
+. image:img/button-complete_cauth.png["The Complete cAuth button", 126,
+  34, role="right"]
+  Complete the chain authorization flow by clicking the **Complete
+  cAuth** button.
++
+The button click dispatches an action to the `templatePrivate` chain and
+updates the state with the shared profile information.
+
+
+=== Additional verification steps
+
+. Verify that the `templatePrivate` chain state has the newly-shared
+profile information. Navigate to the **Block Explorer** and expand the
+`sharedProfile` property.
++
+image:img/screen-template_app-be-shared.png["The Block Explorer screen
+in the Template app, showing that the profile is now shared", 1280,
+659, role="frame"]
++
+The `name` and `email` fields are populated with the values entered from
+the Accounts app.
++
+The `accountsPrivate` and `templatePrivate` chains are now joined.
+Whenever any updates to the user's name or email occur in the Accounts
+app, these changes are automatically reflected in the `templatePrivate`
+chain.
+
+. Verify that the chain join on the `accountsPrivate` chain. From the
+  **Block Explorer** in the Accounts app, expand the `shared` property
+  and check that there is an entry with the same profile identifier as
+  in <<step_authorize,the authorization step>>.
++
+image:img/screen-accounts_app-be-shared-connected.png["The Block
+Explorer screen in the Accounts app, showing the shared profile now
+contains populated data", 1280, 581, role="frame"]
```

Comment:
Could we add a conclusion for this section and the entire platform example? The other sections wrap up nicely with short statements about what was completed in the section and what's to come in the next section. This feels like an abrupt ending to the example. (Sorry, I should have written one in my first attempts at these docs).